src/test/java/io/jenkins/plugins/conventionalcommits/utils/CurrentVersionTest.java

```diff
@@ -155,6 +155,69 @@ public void testNpmProjectVersion() throws IOException, InterruptedException {
     assertThat(actualCurrentVersion, is(testCurrentVersion));
   }
 
+  @Test
+  public void testPythonProjectVersion_SetupPyExists() throws IOException, InterruptedException {
+
+    String os = System.getProperty("os.name");
+    String commandName = "python";
```

Comment:
Not something you need to have for this PR, but you could pull this out to a configuration so the command is the same for the test and for the main class.

Reply:
Do you mean having a `config.java` or something from where I can just import and use the command? (for this and other project types as well?)